e2e-tests/test-applications/nuxt-4-test-app/package.json

```diff
@@ -14,5 +14,8 @@
     "nuxt": "3.15.0",
     "vue": "latest",
     "vue-router": "latest"
+  },
+  "resolutions": {
```

Comment:
do we need this for the test app to run? just curious, no action required

Reply:
Yes, it was prompting and breaking the E2E test flow before adding this resolution